scico/numpy/__init__.py

```diff
@@ -5,163 +5,45 @@
 # user license can be found in the 'LICENSE' file distributed with the
 # package.
 
-""":class:`scico.blockarray.BlockArray`-compatible
+""":class:`scico.numpy.BlockArray`-compatible
 versions of :mod:`jax.numpy` functions.
 
 This modules consists of functions from :mod:`jax.numpy` wrapped to
-support compatibility with :class:`scico.blockarray.BlockArray`. This
+support compatibility with :class:`scico.numpy.BlockArray`. This
 module is a work in progress and therefore not all functions are
 wrapped. Functions that have not been wrapped yet have WARNING text in
 their documentation, below.
 """
```

Comment:
But also some miscellaneous other stuff that was copied in from `scico.array`. Is this module really the best home for this stuff?

Reply:
Probably not. I can put some more thought into what should go where.

Reply:
As discussed, all of this moved into `snp.util` and `snp` has a new docstring.